packages/frontend/src/app/app-component/app.component.html

```diff
@@ -33,6 +33,10 @@
     </mat-sidenav>
 
     <mat-sidenav-content>
+      
+      <!-- BREADCRUMBS -->
```

Comment:
nit: comment might not be needed, since the code below is self documenting ... the problem with comments like this is that we might eventually  decide to use an alternate component, and leave the comment ... future generations will have:

```html
 <!-- BREADCRUMBS -->
 <app-not-a-bread-crumb />
```